parsemsgpack.m

```diff
@@ -62,93 +60,93 @@
     end
 
     switch currentbyte
-        case 192 % nil
+        case uint8(192) % nil
             obj = [];
             idx = idx+1;
-      % case 193 % unused
-        case 194 % false
+      % case uint8(193 % unused
```

Comment:
this seems wrong

Reply:
yes, it is wrong but commented out. I will delete it then.